docker-readme/auto-setup.md

```diff
@@ -90,22 +104,38 @@ This variable specifies your PostgreSQL password.
 
 `POSTGRES_TLS_ENABLED` (defaults to `false`)
 
+This variable specifies whether you are using TLS to connect to Postgres.
+
 `POSTGRES_TLS_DISABLE_HOST_VERIFICATION` (defaults to `false`)
 
+This variable specifies whether Postgres should skip host key verification (e.g., in case your hostnames have changed recently and you need to work around a false positive red flag).
```

Comment:
This is more generally used for server certificates which you can't easily verify, such as Amazon RDS.